site.yml

```diff
@@ -23,6 +23,7 @@
 - hosts: all
   roles:
     - docker
+    - { role: weave, when: weave_enabled|bool }
```

Comment:
sounds good, only concern is how to handle software using weave proxy in a this approach friendly manner